src/__tests__/test-utils-wrapper.test.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ElementWrapper as DomElementWrapper } from '../../lib/components/test-utils/dom';
+import { ElementWrapper as SelectorElementWrapper } from '../../lib/components/test-utils/selectors';
```

Comment:
Did you explore an option of just reading this file and taking it whole as a snapshot?

Reply:
If you heard "snapshot testing is bad", it was about React testing.

For code generation snapshot testing is the correct way to test. We do in fact have some snapshot tests for test-utils already: https://github.com/cloudscape-design/test-utils/blob/main/src/converter/test/__snapshots__/converter.test.ts.snap

So, adding more tests like this would make sense

Reply:
We should be keeping the contract for TS, so we should expect no API signature change. Aside from the function names, this would only control the comments but it would be sensitive to the whitespace changes.

I think we can get more value if we write assertions for each function rather than a simple snapshot test, but it might be also a bit slower.

e.g.
```tsx
jest.mock('@cloudscape/test-utils')

test.each(functionsInWrapper)(func => {
  func(); 
  expect(testUtils.findAllComponents).toHaveBeenCalledWith(CardWrapper.rootSelector)
})
```

Reply:
To update this thread:

1. Snapshot tests are not performed on the entire file.
2. I added a few functional tests to make sure that the statically generated files are passing the params correctly.
3. We put the `pluralize.js` out and used a static map instead.